applications/luci-app-wireguard/Makefile

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=WireGuard Status
+LUCI_DEPENDS:=+wireguard
```

Comment:
Done!
This applies to luci-proto-wireguard as well, then, right? So I better add this line there as well?